### DISCUSSION TITLE
ServerlessError: No file matches include / exclude patterns

### DISCUSSION THREAD

**Discussion by nallurisrinicbs** [serverless/serverless#10073] [Q&A]
Hello,

I am getting this error "ServerlessError: No file matches include / exclude patterns" at xxxxxxxxxxxxx\node_modules\serverless\lib\plugins\package\lib\packageService.js:301:13

Please help!

**Comment by pgrzesik** [comment]
Hello, please include more details about your problem, ideally with a simple reproducible example. Otherwise it's really hard to help as the above information does not provide too much context.

**Reply by nallurisrinicbs** [reply]
My coworkers are not getting this error on their computer for the same code. So this error is only on my computer.  I get this error for this command: serverless deploy --stack teststackname

**Reply by pgrzesik** [reply]
As I mentioned previously, could you please provide a reproducible example with `serverless.yml` that I could check out on my side? Otherwise it's pretty much impossible to help in any capacity

**Reply by nallurisrinicbs** [reply]
I am good now, thanks. I don't know exact reason why it didn't work before!

**Reply by pgrzesik** [reply]
Happy to hear you've been able to resolve your issue :raised_hands:
